### PR TITLE
Remove ERP Verbs 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by Chaznoodles**
We've been over this a lot. Rhicora's attempted PR got shot down by people whining, so I'm going to lay out a few reasons off the bat why removing the verbs is a good change.

**ERP verbs aren't conductive at all to a good server.**
First off the bat, it draws a bad crowd from Citadel and other sub-par servers. These people and their attitudes towards others tend to push people who might be interested in roleplay in a Fallout universe away, as well as creating a bad reputation for the server in general. Removing ERP would remove this stigma making BD citadel-tier, thereby making it more likely to attract and retain prospective players.
One common response to this is "But the server will die without ERP!". My response to that is: If it dies without ERP, then let it die. However, the population dropoff as a result of removing ERP - with the Citadel players and people who only come here to ERP verb leaving - will be negated by the actual interest of roleplayers and average SS13 players with an interest in the subject and a change of pace in gameplay.

**Removing ERP verbs will result in a net improvement in server quality.**
When you remove ERP verbs, as per the above, you remove the players who are soley interested in it and their attitudes towards others. This results in an increase in player quality, from the retained players who are here for the Fallout atmosphere and roleplay, and the new players with the same interests. As a result of this, staff quality will increase as there's a better quality pool of players to draw from. This results in rules getting enforced more uniformly, a better role model for newer players, and a higher quality of community management. Together, these factors will draw more players in and create a looping effect of better players = better staff = better server.

If you like ERP verbs and want them to stay, I'd ask you to respond to a few questions instead of just getting whiny and downvoting:
1. Why do ERP verbs contribute to making BD a good/quality server?
2. In what way do they contribute?
3. Do you also play on other servers with ERP verbs enabled?

Happily answer other questions if they're posed as to why ERP should be removed. Exactly the same changes Rhicora made.

ahurepm

### CODE REVIEW

**Comment by Kimochiiiiiiiiiiiiii** [comment]
**Why do ERP verbs contribute to making BD a good/quality server?**
ERP verbs contribute to any round by allowing raiders to do inmoral actions, rape, and creating a path on the Flow of the round by doing it, maybe by causing a local militia setting up, maybe a faction taking over and fighting them while taxing the victims-to-be.
**In what way do they contribute?**
The ways that it can contribute in the round is obviously in the scenario that two players have set up a specific roleplay as a family or spouse/husband relations and want to consume it ICly without doing a whole /me. The other is just what I said above, raider roleplay. By that I mean it being used correctly, not going straight to someone, copy pasting two shitty lines and raping them with throat fucking after buckshotting them in the face.
**Do you also play on other servers with ERP verbs enabled?**
Fuck no lmao

**Comment by UndeadNappist** [comment]
#332 we already had most of the discussion already, don't cover old ground.

**Comment by imw-Oculus** [comment]
Very well, let's do this entire rodeo again. To start with - why did you make an identical PR? Is this going to be a case where it's just going to keep getting pushed until it finally goes through? 

I'm going to go through this assuming the burden of proving a reason for the removal lies more on your shoulders than it does on the people who want to keep it in, since the previous PR was shot down.

"First off the bat, it draws a bad crowd from Citadel and other sub-par servers."

Do you have any actual evidence of this?

"These people and their attitudes towards others tend to push people who might be interested in roleplay in a Fallout universe away,"

What attitudes would those be, specifically? I would additionally argue that sex and rape are an integral part of the Fallout universe - whitewashed Bethesda games aside.

"as well as creating a bad reputation for the server in general."

This PR isn't for removing ERP, it's for removing ERP verbs. The reputation of the server wouldn't change one iota. I would also argue that - based on player count - regardless of our reputation, we enjoy a decent number of players. I don't want to see anything that impacts that implemented.

"Removing ERP would remove this stigma making BD citadel-tier, thereby making it more likely to attract and retain prospective players."

Do you have any actual evidence that this would be the case?

"One common response to this is "But the server will die without ERP!". My response to that is: If it dies without ERP, then let it die."

I don't even know how to respond to this. You would rather have the server die than there be ERP verbs? Again - to be clear - this PR doesn't remove ERP, it removes ERP verbs. If you don't like an element that is integral to the server, then leave?

"However, the population dropoff as a result of removing ERP - with the Citadel players and people who only come here to ERP verb leaving - will be negated by the actual interest of roleplayers and average SS13 players with an interest in the subject and a change of pace in gameplay."

Where are these projections coming from? You make a bunch of assertions about what will happen to our player count, but I don't see any actual evidence to support it. I suspect a couple people would leave, yes, but I hugely doubt there would be some magical dawning of a new era because of a single relatively minor change.

"When you remove ERP verbs, as per the above, you remove the players who are solely interested in it and their attitudes towards others."

I can't argue that removing ERP verbs would remove people who are only here for ERP verbs, yes.

"This results in an increase in player quality, from the retained players who are here for the Fallout atmosphere and roleplay, and the new players with the same interests."

While we're making projections - allow me to submit one of my own. The number of players who are here SOLELY to mash ERP buttons are in a minority. There are players who do that as a part of the activities in which they partake, but I would argue that the people who actually are here for the express purpose of pushing ERP buttons are in an extreme minority. ERP verbs enhance the experience of the game. They are a seasoning. They are not the main course.

"As a result of this, staff quality will increase as there's a better quality pool of players to draw from."

I hope you're not insinuating that our staff are poor.

 "This results in rules getting enforced more uniformly, a better role model for newer players, and a higher quality of community management. Together, these factors will draw more players in and create a looping effect of better players = better staff = better server."

I'm getting the distinct impression that you aren't a fan of the staff.

----

tl;dr - You have a lot of projections. They're great projections. If everything went exactly like you said it would, then yes - I can see how removing ERP verbs would be a positive change in your eyes. Here is a counter-projection for you. Removing ERP verbs - which are, as I stated above, are more of a seasoning than the main course - would have relatively little impact. A few people would get pissed off and leave. That's probably all that would happen.  No sudden flood of new players. No dawn of a new era. This entire PR reads like you're trying to remove ERP as a whole, instead of just ERP verbs. I also very much resent you making an identical PR to one that was already submitted, and shot down.

Oh, and I don't see anyone enjoying a half an hour of getting text-raped, as opposed to it being over in about 30 seconds at most.

Very tl;dr - Removes a feature. I am against removing features, unless there is a reason to remove them. There is no reason to remove ERP verbs that isn't based entirely on conjecture.
